docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -34,69 +34,61 @@ Include it in your module file like this:
+  The name of the module. This must be the same as the filename, without the ``.py`` extension.
 * ``short_description:``
 
-  * Displayed on the :doc:`../list_of_all_modules` page and ``ansible-doc -l``.
-  * Must have enough detail without the context of the category.
-  * Without a trailing full stop
+  * A short description which is displayed on the :doc:`../list_of_all_modules` page and ``ansible-doc -l``.
+  * As the short description is displayed by ``ansible-doc -l`` without the category grouping it needs enough detail to explain it's purpose without the context of the directory structure in which it lives.i
+  * Unlike ``description:`` this field should not have a trailing full stop.
 * ``description:``
-  * A longer description than `short_description`, which goes into more detail, generally two or more sentences.
+  * A detailed description (generally two or more sentences).
   * Must be written in full sentences, i.e. with capital letters and fullstops.
   * Shouldn't mention the name module.
 * ``version_added:``
   The version of Ansible when the module was added.
   This is a `string`, and not a float, i.e. ``version_added: "2.1"``
 * ``author:``
-  In the form ``First Last (@GitHubID)``, use a multi-line list if there is more than one.
+  Name of the module author in the form ``First Last (@GitHubID)``. Use a multi-line list if there is more than one author.
 * ``options:``
-  One per option name
+  One per module argument
 
   * ``description:``
 
-    * Detailed explanation of what this option does and what it does. It should be written in full sentences.
-    * Should not list the option values (that's what ``choices:`` is for, though it should explain `what` the values do if they aren't obvious.
-    * If an argument takes both True/False and Yes/No, the documentation should use True and False.
+    * Detailed explanation of what this option does. It should be written in full sentences.
+    * Should not list the options values (that's what ``choices:`` is for, though it should explain `what` the values do if they aren't obvious.
+    * If an argument takes both True)/False and Yes)/No, the documentation should use True and False.
     * If an optional parameter is sometimes required this need to be reflected in the documentation, e.g. "Required when I(state=present)."
     * Mutually exclusive options must be documented as the final sentence on each of the options.
   * ``required:``
     Only needed if true, otherwise it is assumed to be false.
   * ``default:``
     * If `required` is false/missing, `default` may be specified (assumed 'null' if missing).
-    * Ensure that the default parameter in docs matches default parameter in code. The default option must not be listed as part of the description.
-  * ```choices``
+    * Ensure that the default parameter in the docs matches default parameter in the code. The default option must not be listed as part of the description.
```

Comment:
Please change to: "Ensure that the default parameter in the docs matches the default parameter in the code. The default option must not be listed as part of the description."